fvm/environment/account_creator.go

```diff
@@ -0,0 +1,108 @@
+package environment
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const keyAddressState = "account_address_state"
+
+type AccountCreator struct {
```

Comment:
nit: could you also move over the original comments for StateBoundAddressGenerator?

Reply:
that comment no longer makes sense once I've merge TransactionEnv.CreateAccount into this.  I'll write some else up

Reply:
added comment